ForgeUnity/Assets/BeardedManStudios/Scripts/Networking/Forge/Networking/Objects/NetworkObject.cs

```diff
@@ -208,6 +208,17 @@ public abstract class NetworkObject
 		/// </summary>
 		public NetworkingPlayer Owner { get; private set; }
 
+		/// <summary>
+		/// A reference to the player who can send RPCs this network object
+		/// </summary>		
+		public NetworkingPlayer Master { get; private set; }
```

Comment:
How does this change handle things when a NetworkingPlayer is destroyed after a disconnection?
Wondering about the effect of having a network object still being used temporarily with a `Master` set to null, for which the code would assume the object is not Master on the local client.

Having a bool IsMaster (which is initially set in AssignMaster) would prevent losing the information that the object is Master on the local client, even if the Master reference becomes null.

Reply:
I agree with the above

Reply:
On local clients this would either be their own Networking player if they are the master or the server as they only know about these two. This would only be set to different client's networking player on the server which know all clients.